### DISCUSSION TITLE
how to style the workspaces before and after of the current workspace?

### DISCUSSION THREAD

**Discussion by laraia231** [Alexays/Waybar#2246] [Q&A]
i'd like to know if this is possible, i'm pretty new to linux and ricing in general so im not really sure. 

if the title isn't that clear, let's say the current workspace i'm currently in is 4, i want to style workspace #3 and workspace #5. 

unless ofc, it's a workspace in the corner wherein that case it, i would be styling either before or after the current workspace.

**Comment by lukejcollins** [comment]
Hi there. I'd like to help answer this but the question is a little unclear. Could you maybe reword it slightly? Or provide a mock-up image of what you're looking for? Thanks 🙂
